### PR TITLE
Added info about tsql_utils to README

### PR DESCRIPTION
**Pull request by infused-kim**
The first time i tried to use dbt-sqlserver, it was somewhat confusing and difficult to figure out how to make dbt-utils work.

Hopefully this will make it easier for new users.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -155,6 +155,11 @@ client_secret: clientsecret
 
 But, columns in source table can not have any constraints. If for example any column has a NOT NULL constraint, an error will be thrown.
 
+### DBT Utils
+Many DBT utils macros are supported, but they require the addition of the `tsql_utils` dbt package. 
+
+You can find the package and installation instructions in the [tsql-utils repo](https://github.com/dbt-msft/tsql-utils).
```
Comment (swanderz) [inline README.md:161]:
```suggestion
Check out [dbt-msft/tsql-utils](https://github.com/dbt-msft/tsql-utils) for installation instructions
```

**Review by swanderz — APPROVED** [review]
thanks @infused-kim, any chance you'd be willing to do the same for [dbt-msft/dbt-synapse](https://github.com/dbt-msft/dbt-synapse)?

@mikaelene reviewing this made me think about the entire "Supported Features", I'm not sure it needs to exist in it's current state. Perhaps just a section that notes how dbt-sqlserver works differently than dbt-core (e.g., data tests with CTEs in them are broken)?

**Comment by infused-kim** [comment]
THanks for merging it in, guys!


> thanks @infused-kim, any chance you'd be willing to do the same for [dbt-msft/dbt-synapse](https://github.com/dbt-msft/dbt-synapse)?

Sure, I would be more than happy to do that, but is it necessary? I saw that it mentions tsql-utils already:
> dbt-msft/tsql-utils enables dbt-synapse to use dbt-utils: the much-loved, extremely-useful collection of dbt macros.

**Comment by swanderz** [comment]
> but is it necessary? I saw that it mentions tsql-utils already
whoops -- you're right!

**Pull request merged**